data/json/recipes/recipe_others.json

```diff
@@ -3391,6 +3391,22 @@
     "tools": [ [ [ "surface_heat", 15, "LIST" ] ] ],
     "components": [ [ [ "milk_standard_raw", 15, "LIST" ] ], [ [ "bleach", 1 ], [ "vinegar", 4 ] ] ]
   },
+  {
+    "type": "recipe",
+    "id_suffix": "bioplastic",
+    "result": "plastic_chunk",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "chemistry",
+    "difficulty": 3,
```

Comment:
Level 3 chemistry is an upper level of "kids' chemistry". Is this new recipe doable for kids? Just to be sure.

Reply:
It's a really simple recipe IRL. https://www.youtube.com/watch?v=5M_eDLyfzp8